TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/Factories/Shifter/Shifter.as

```diff
@@ -0,0 +1,59 @@
+﻿
+void onInit(CBlob @ this)
+{
+
+	this.set_TileType("background tile", CMap::tile_castle_back);
+
+	this.getShape().SetRotationsAllowed(false);
+
+	this.Tag("builder always hit");
+
+	this.set_u32("charge",0);
+	CSprite@ sprite = this.getSprite();
+	sprite.SetAnimation("off"); //Starts offline
+	sprite.SetZ(-100.0f);
+}
+
+void onTick(CBlob@ this)
```

Comment:
Optimization: Remove onTick and move the logic into onCollosion

Store last time used and if it's more > 120 ticks then push the blob


Put sprite animation into onTick(CSprite@)  so only clients will run that code